packages/lbvs_consent/lbvs_consent.1.1.2/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
```

Comment:
Is it intentional that you use 2.0 here? This looks like a 1.2 package to me, have you checked that it works with opam2?

Reply:
I use opam 2.0.0~beta5.

Reply:
But the present repository is still in opam1 format. I don't know how to check that it also works for opam2 (I suspect it doesn't, because the syntax has changed a bit; your opam2 installation may be silently converting from one to the other), and I would suggest using `1.2` as all other packages before the repository is converted in bulk.